manifests/m/Musescore/Musescore/4.2.1.240230937/Musescore.Musescore.locale.en-US.yaml

```diff
@@ -0,0 +1,35 @@
+# Created with YamlCreate.ps1 v2.2.13 $debug=QUSU.LF.7-4-0.Unix
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Musescore.Musescore
+PackageVersion: 4.2.1.240230937
+PackageLocale: en-US
+Publisher: MuseScore BVBA and Others
+PublisherUrl: https://musescore.org/en
+PublisherSupportUrl: https://musescore.org/en/support
+PrivacyUrl: https://musescore.com/legal/privacy
+Author: Musescore
+PackageName: MuseScore
+PackageUrl: https://musescore.org/en
+License: GPL-3.0 License
+LicenseUrl: https://github.com/musescore/MuseScore/blob/master/LICENSE.txt
+Copyright: Copyright (c) 1999-2023 MuseScore BVBA and others
+CopyrightUrl: https://musescore.com/legal/dmca
```

Comment:
```suggestion
# PrivacyUrl: 
Author: Musescore
PackageName: MuseScore
PackageUrl: https://musescore.org/en
License: GPL-3.0 License
LicenseUrl: https://github.com/musescore/MuseScore/blob/master/LICENSE.txt
Copyright: Copyright (c) 1999-2023 MuseScore BVBA and others
# CopyrightUrl: 
```
The host might have an IP block preventing our data center from connecting, to prevent automated downloads.

Reply:
Thanks, @stephengillie. I have noticed that `curl -I https://musescore.com` also returns a 403.